### ISSUE TITLE
Bibliography and SVGs

### ISSUE THREAD

**Issue by ishanina** [michal-h21/make4ht#31]
Hi, I am trying to use make4ht to convert a math document to html. I have the latest version of everything and am running the following command:

`make4ht Descent.tex "mathml,mathjax"
`
 
There are three issues right with the output now.

1. The bibliography is not coming out at all.
2. The diagrams are not coming out. When I open the resulting html document, in the place that the SVGs should be, I see 
`Unknown node type: object`
in yellow, followed by 
` SVG-Viewer needed.`
I am using Chrome as my browser.
3. The SVGs for the diagrams aren't being created correctly. They are diagrams made using TikZ-cd. When I open the SVGs, I can see the arrows, but not the nodes in the diagram.

Do you have any advice on how to fix these? Thanks!

**Comment by ishanina** [comment]
I was able to solve 1. by following the instructions to your answer here: https://tex.stackexchange.com/questions/358565/can-htlatex-be-used-with-biblatex-these-days

However, 2. and 3. are still problems for me.

**Comment by ishanina** [comment]
I *finally* got it to work. In order to do so, I used the driver from [here](https://github.com/michal-h21/dvisvgm4ht), which I became aware of from [this](https://tex.stackexchange.com/questions/543919/what-are-the-limitations-of-using-tikz-with-tex4ht). However there were a few other issues I had to fix along the way:

My version of dvisvgm was not working with postscript. The eventual solution was to reinstall dvisvgm using MacPorts (trying to directly compile the binary was unsuccessful).

Then, there were fonts that were missing, which again MacPorts was able to help install. After that everything worked beautifully! Thanks so much for this great package!
